cmake/modules/SwiftList.cmake

```diff
@@ -30,3 +30,12 @@ function(list_union lhs rhs result_var_name)
   endforeach()
   set("${result_var_name}" "${result}" PARENT_SCOPE)
 endfunction()
+
+function(list_contains list item result_var_name)
+  list(FIND list "${item}" index)
+  if(${index} EQUAL -1)
+    set("${result_var_name}" 0 PARENT_SCOPE)
+  else()
+    set("${result_var_name}" 1 PARENT_SCOPE)
+  endif()
+endfunction()
```

Comment:
We've raised our minimum-supported CMake since I pushed this, so I think it can be replaced with a built-in function now anyway